Cesium.Runtime/StringFunctions.cs

```diff
@@ -14,50 +12,57 @@ public static unsafe class StringFunctions
 {
     public static nuint StrLen(byte* str)
     {
-#if NETSTANDARD
-        if (str == null)
+        if (str != null)
         {
-            return 0;
-        }
+            var start = str;
+            while ((nuint)str % 16 != 0)
```

Comment:
I suggest introducing a constant for this `16`, like `MinimalPageBoundary` or something, to make the size choice more clear.

Reply:
Right, it's just `Vector128<byte>.Count` which I omitted because surrounding code is compact too, let me fix it. (to which we align - this is both for performance and for never crossing page boundary, whatever size it is, it occured to me we don't need actual page size here since it's always bigger than 16B and 16B alignment will never cross it)